composer.json

```diff
@@ -18,11 +18,11 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/event-dispatcher": "~2.3"
+        "symfony/event-dispatcher": "~2.3|~3.0",
+        "guzzlehttp/guzzle": "^6.2.0"
```

Comment:
One of the features of BaseApi is to be "http client agnostic". This is against the philosophy of NeklandBaseApi.